packages/api/src/task/scheduler.ts

```diff
@@ -244,22 +246,27 @@ export default class TaskScheduler {
       ...updates,
     };
     const timestamp = task.status.updatedAt;
-    await this.queue.publishWebhook("events.task.status", {
+    await this.queue.publishWebhook("events.task.updated", {
       type: "webhook_event",
       id: uuid(),
       timestamp,
-      event: "task.status",
+      event: "task.updated",
       userId: task.userId,
       payload: {
         task: taskInfo(task),
       },
     });
     if (task.status.phase === "completed" || task.status.phase === "failed") {
-      await this.queue.publishWebhook("events.task.finished", {
+      let task_event: EventKey = "task.succeeded";
```

Comment:
Please use `camelCase` for any variable names as we should keep a consistent style in the code.

We should have a linter for this, but the standard in the project is `camelCase` everywhere unless maybe some static `UPPERCASE_CONSTANTS` in the root of the file.